src/test/java/com/bbtutorials/users/dao/WorkoutDaoTests.java

```diff
@@ -0,0 +1,133 @@
+package com.bbtutorials.users.dao;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.local.embedded.DynamoDBEmbedded;
+import com.amazonaws.services.dynamodbv2.model.*;
+import com.bbtutorials.users.model.Workout;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.Assert.*;
+
+public class WorkoutDaoTests {
+    private static AmazonDynamoDB amazonDynamoDB;
+    private static DynamoDBMapper mapper;
+    private static WorkoutDao workoutDao;
+
+
+    private final String testEmail = "test@mail.com";
+    private final String testName = "Name";
+    private final String testId = "test";
+
+    @BeforeClass
+    public static void setup() {
+        System.setProperty("sqlite4java.library.path", "native-libs");
+
+        amazonDynamoDB = DynamoDBEmbedded.create().amazonDynamoDB();//dynamoDB.getAmazonDynamoDB();
+        mapper = new DynamoDBMapper(amazonDynamoDB);
+
+        amazonDynamoDB.createTable(new CreateTableRequest()
+                .withTableName("Workouts")
+                .withKeySchema(new KeySchemaElement().withAttributeName("Id").withKeyType(KeyType.HASH))
+                .withAttributeDefinitions(
+                        new AttributeDefinition().withAttributeName("Id").withAttributeType(ScalarAttributeType.S))
+                .withProvisionedThroughput(new ProvisionedThroughput(5L, 5L))
+        );
+
+        workoutDao = new WorkoutDao(mapper);
+    }
+
+    public Workout createWorkout() {
+        Workout testWorkout = new Workout();
+        testWorkout.setId(testEmail);
```

Comment:
The ID for workouts is auto-generated, like in exercises, so you don’t set it to anything